src/WTG.Analyzers/Rules/Rules.xml

```diff
@@ -231,6 +231,21 @@
 			<description>Combining bool expressions in larger boolean expressions makes the code more complex than it needs to be.</description>
 			<tag>Unnecessary</tag>
 		</rule>
+		<rule id="13" name="DontUseConcatWhenAppendingSingleElementToEnumerables" severity="Warning">
```

Comment:
`Warning` feels a little heavy-handed for these, I wonder if they should be `Info` instead.